xml/app_uaa_values_yaml.xml

```diff
@@ -31,11 +32,15 @@ kube:
   # Whether HostPath volume mounts are available
   hostpath_available: false
 
-  organization: "cap"
+  limits:
+    nproc:
+      hard: ""
+      soft: ""
+  organization: "splatform"
   psp:
     nonprivileged: ~
   registry:
-    hostname: "registry.suse.com"
+    hostname: "staging.registry.howdoi.website"
```

Comment:
Is the the real literal hostname? we're using howdoi.website?

Reply:
I'll fix that and refrain from copy pasting till the file is finalized.